.nvmrc

```diff
@@ -0,0 +1 @@
+14.8.0
```

Comment:
We've been getting into the habit of being less specific and setting these files to value such as `14` as that has proven to be good enough for our needs.